### ISSUE TITLE
Refuse to start if bpffs filesystem is mounted multiple times

### ISSUE THREAD

**Issue by tgraf** [cilium/cilium#1257]
Add relevant logic to:
```
        _, err := exec.Command("mountpoint", args...).CombinedOutput()
        if err != nil {
                args = []string{"bpffs", mapRoot, "-t", "bpf"}
                out, err := exec.Command("mount", args...).CombinedOutput()
```

**Comment by eloycoto** [comment]
I fix this issue on PR #1460

**Comment by manalibhutiyani** [comment]
@Eloy, your change fixed part of the problem, i.e deny if the mounted fs is not bpf. This change tracks that if the filesystem is already mounted, is it mounted more than once, and if yes, it fails the daemon.
